flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/FlinkService.java

```diff
@@ -93,4 +93,6 @@ void triggerSavepoint(
     PodList getJmPodList(FlinkDeployment deployment, Configuration conf);
 
     void waitForClusterShutdown(Configuration conf);
+
+    boolean reactiveScale(ObjectMeta meta, JobSpec jobSpec, Configuration conf);
```

Comment:
I think we should simply call this `scale` . This operation does not to be reactive, that's basically just one way to implement this for standalone.

Reply:
It could also return false by default to avoid having to implement it for non standalone clusters.